### ISSUE TITLE
Missing package.json

### ISSUE THREAD

**Issue by tecfu** [GoThinkster/conduit-bootstrap-template#4]
Can't run 

```
npm install
```
without a package.json now, can we?

**Comment by amrit123456789** [comment]
yeah..having trouble on how to integrate this template with backend..someone please help!

**Comment by cirosantilli** [comment]
Added one at https://github.com/gothinkster/conduit-bootstrap-template/pull/6/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519 based on: https://github.com/gothinkster/conduit-bootstrap-template/pull/2/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519 but working with newer node.
